config/controller/namespace.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    app.kubernetes.io/name: argocd-ephemeral-access
+    app.kubernetes.io/managed-by: kustomize
+  name: system
```

Comment:
That is a strange name for a namespace. What is the intent with it?

Reply:
namespace set in kustomize: `argocd-ephemeral-access`
but the k8s resource needs a name as a placeholder

Reply:
in this case, can you change it to `name: placeholder` to avoid this confusion?

Reply:
That would be inconsistent with what kubebuilder is setting in the `rbac/` folder